packages/wrangler/src/sourcemap.ts

```diff
@@ -118,7 +120,7 @@ export function getSourceMappedString(value: string): string {
  */
 
 const CALL_SITE_REGEXP =
-	/at (?:(.+?)\s+\()?(?:(.+?):(\d+)(?::(\d+))?|([^)]+))\)?/g;
+	/ {4}at (?:(.+?)\s+\()?(?:(.+?):(\d+)(?::(\d+))?|([^)]+))\)?/g;
```

Comment:
IMHO this entire 'feature' wants to be scrapped. No way can it be correct to perform REGEX on the output like this to replace console log messages. This is just ASKING for endless weird bugs and frustration.

Basically this just seems to hide the bug and change it from 'at' to ' {4}at' which is also quite likely to be a normal console log message a user might want to log.